ui-tests/src/test/resources/features/integrations/integration-import-export.feature

```diff
@@ -62,16 +54,14 @@ Feature: Integration - Import Export
     And click on the "Publish" button
     # assert integration is present in list
     Then check visibility of "Integration_import_export_test" integration details
+    And navigate to the "Integrations" page
 
-    When navigate to the "Integrations" page
-    Then Integration "Integration_import_export_test" is present in integrations list
+    And Integration "Integration_import_export_test" is present in integrations list
     # wait for integration to get in active state
+    And sleep for "300000" ms
```

Comment:
This is not even imported integration so if the wait is necessary here, something is wrong with db to slack integration (not started in 10 mins = performance issue, please report it)